doc/user_doc/vic_installation/set_up_ops_user.md

```diff
@@ -0,0 +1,72 @@
+
+5. In each of the **Hosts and Clusters**, **Storage**, and **Networking** views, select inventory objects and assign the user group and the appropriate role to each one.
+
+   [Inventory Objects to Which to Assign Permissions](#inventory_objects) below lists the inventory objects that VCHs access, and the role that you must assign to each one.
+
+**What to do next**
+
+Use `vic-machine create` to deploy VCHs, specifying an appropriate user account in the `--ops-user` option. Specify the inventory objects on which you have set permissions in the various `vic-machine create` options. For an example of a `vic-machine` command with the `--ops-user` option, see the section *Specify Different User Accounts for VCH Deployment and Operation* [Advanced Examples of Deploying a VCH](vch_installer_examples.md#ops-user).
+
+<a name="required_roles"></a>
+## Required VCH Roles ##
+
+You must create the following vSphere roles for use by the `--ops-user` account.
+
+|**Role to Create**|**Required Permissions**|
+|---|---|
+|`VCH - vcenter`|Datastore &gt; Configure datastore|
+|`VCH - datacenter`|Datastore &gt; Configure datastore<br>Datastore &gt; Low level file operations|
+|`VCH - datastore`|Datastore &gt; AllocateSpace<br>Datastore &gt; Browse datastore <br>Datastore &gt; Configure datastore<br>Datastore &gt; Remove file<br>Datastore &gt; Low level file operations<br>Host &gt; Configuration &gt; System management|
+|`VCH - network`|Network &gt; Assign network|
+|`VCH - endpoint`|dvPort group &gt; all permissions<br>vApp &gt; Add virtual machine<br>VirtualMachine &gt; Configuration &gt; Add new disk<br>VirtualMachine &gt; Configuration &gt; Advanced<br>VirtualMachine &gt; Configuration &gt; Add or Remove Device<br>VirtualMachine &gt; Configuration &gt; Remove disk<br>VirtualMachine &gt; Guest operations &gt; Guest operation program execution<br>VirtualMachine &gt; Interaction &gt; Device connection<br>VirtualMachine &gt; Interaction &gt; Power off<br>VirtualMachine &gt; Interaction &gt; Power on<br>VirtualMachine &gt; Inventory &gt; Create new<br>VirtualMachine &gt; Inventory &gt; Remove<br>VirtualMachine &gt; Inventory &gt; Register<br>VirtualMachine &gt; Inventory &gt; Unregister|
+
+<a name="inventory_objects"></a>
+## Inventory Objects to Which to Assign Permissions ##
+
+You must assign permission to perform the VCH roles to the appropriate objects in the vCenter Server inventory.
+
+|Inventory Object|Role to Assign|Propagate|
+|---|---|---|
+|Top-level vCenter Server instance|`VCH - vcenter`|No|
+|Datacenters|`VCH - datacenter`|No|
+|Clusters. All datastores in the cluster inherit permissions from the cluster.|`VCH - datastore`|Yes|
+|Standalone VMware vSAN datastores|`VCH - datastore`|No|
+|Standalone datastores|`VCH - datastore`|No|
+|Network folders|`Read-only`|Yes|
+|Port groups|`VCH - network`|No|
+|Resource pools|`VCH - endpoint`|Yes|
```

Comment:
Added a new row to that table that includes "VCH vApps, for a very granular application of privileges". Expanded the description of the step about creating a resource pool:

4. (Optional) Go to **Hosts and Clusters** and create a resource pool in which to deploy VCHs.

   By creating a resource pool for VCHs, you can set the correct permissons on just that resource pool rather than on an entire host or cluster. You specify this resource pool in the `vic-machine create --compute-resource` option when you deploy the VCH. For a more granular application of privileges, you can also apply the `VCH - endpoint` role directly to VCH vApps after deployment, rather than to a resource pool.